zarf.yaml

```diff
@@ -1,14 +1,25 @@
 kind: ZarfInitConfig
+metadata:
+  name: Zarf Official Init Package
+  description: "Used to establish a new Zarf cluster"
+
+seed:
+  - library/registry:2.7.1
 
 components:
   - name: k3s
     description: >
+      *** REQUIRES ROOT ***
       Install K3s, certified Kubernetes distribution built for IoT & Edge computing.
       K3s provides the cluster need for Zarf running in Appliance MOde as well as can
       host a low-resource Gitops Service if not using an existing Kubernetes platform.
-    required: true
+    default: true
     scripts:
       retry: true
+      before:
+        # If running RHEL variant, disable firewalld
+        # https://rancher.com/docs/k3s/latest/en/advanced/#additional-preparation-for-red-hat-centos-enterprise-linux
+        - "[ -e /etc/redhat-release ] && systemctl disable firewalld --now"
```

Comment:
If not running a RHEL variant this hangs and eventually makes the init fail because of a timeout.

Reply:
Ooooh It's because of the retries. It's taking the `1` response when checking if the redhat-release file exists as a bash error

Reply:
I resolved this. @jeff-mccoy let me know if you don't like this approach.